src/JJMasterData.Core/DataManager/FormFileContent.cs

```diff
@@ -6,11 +6,7 @@ namespace JJMasterData.Core.DataManager
     public class FormFileContent
     {
         public string FileName { get; set; }
-
-        public MemoryStream FileStream { get; set; }
-
-        public long SizeBytes { get; set; }
-
+        public byte[] Bytes { get; set; }
```

Comment:
Storing at Stream in a object is really bad. **Importation was not working on .NET 6/7.**
https://stackoverflow.com//questions/39962795/storing-a-file-as-a-byte-array-or-a-stream-in-an-object#answer-39962884